CuratedContent/SoftwareSustainabilityMatrix.md

```diff
@@ -0,0 +1,81 @@
+<!--deck text end-->
+
+#### Contributed by [Roscoe A. Bartlett](https://github.com/bartlettroscoe "Roscoe A. Bartlett GitHub Profile")
+#### Publication date: ???
+
+Resource information | Details
+:--- | :--- 
+Article title |  [How Sustainable is Your Software?](https://www.kitware.com/how-sustainable-is-your-software/)
+Authors | Will Schroeder, Jess Tate, Jean-Christophe Fillion-Robin, Christopher Johnson, Dženan Zukić, Matt McCormick, Lee A. Newberg, Daniel White, Alexandra Warner, Ross Whitaker and Rob MacLeod
+Publication Details | [Kitware Blog](https://www.kitware.com/blog), January 3, 2020
+Focus | Software sustainability
+
+The Software Sustainability Matrix (SSM) is a joint effort of the Scientific Computing and Imaging (SCI) Institute and Kitware to improve the health of the open-source package ecosystem making up the foundational software portfolio for the Center for Integrative Biomedical Computing (CIBC) lead by the SCI for the National Institutes of Health (NIH) <sup>[1],[2]</sup>.
+The goal of the SSM effort is to create numerical scores for different aspects of software sustainability for a software package as well of a combined sustainability score.
+These SSM sustainability scores are being to guide efforts to improve the sustainability of the software package ecosystem making to CIBC software portfolio, with the hopes that it will encourage support and contributions from the open-source community and reduce the cost and risks to sustain this collection software.
+However, the SSM approach may be applicable to other package ecosystems and therefore may be of more general interest.
+
+The SSM score is composed from component scores in the four areas Impact, Risks, Community, and Technology as shown in [Figure 1](#fig_software_sustainability_matrix) and each of these areas are broken down into several different subareas.
+
+<a name="fig_software_sustainability_matrix"/>
+
+<br>
+<center>
+<img src='../images/SoftwareSustainabilityMatrix.png' class='page'/>
+</center>
+<br>
+
+<center>
+<b>Figure 1:</b> Software Sustainability Matrix
+</center>
+
+<br>
+<br>
+ 
+Scores in the range [0-100] in these four areas Impact, Risks, Community, and Technology are then combined in a weighted sum:
+
+&nbsp;&nbsp;&nbsp;&nbsp;SSM Score = F<sup>I</sup> * I + F<sup>R</sup> * R + F<sup>C</sup> * C + F<sup>T</sup> * T
+
+to produce a single SSM Score [0-100], where F<sup>I</sup> + F<sup>R</sup> + F<sup>C</sup> + F<sup>T</sup> = 1.
+The SCI & Kitware process<sup>[3]</sup> typically uses the weighting factors F<sup>I</sup> = F<sup>C</sup> = 1/3, and F<sup>R</sup> = F<sup>T</sup> = 1/6.
+It is not specified how the SCI & Ktiware program computes the component scores in the four areas 'I', 'R', 'C', and 'T' for the software packages of interest and the authors state that computing these individual component scores is ultimately a subjective matter.
```

Comment:
```suggestion
It is not specified how the SCI & Kitware program computes the component scores in the four areas 'I', 'R', 'C', and 'T' for the software packages of interest, and the authors state that computing these individual component scores is ultimately a subjective matter.
```

Reply:
I applied the suggested change in commit e325da88e801036e9a5daf42009a94239c9f5437.